docs/patterns/components/Layout/index.js

```diff
@@ -5,15 +5,15 @@ export default function Layout() {
   return (
     <div>
       <h1>Layout</h1>
-      <p>There are 2 different ways to configure a cauldron-react layout.</p>
+      <h2>Component Description</h2>
       <p>
-        It is important to note that the layout components{' '}
+        The Workspace component is a convenience wrapper around the Layout and
+        Main components. It is important to note that the layout components{' '}
         <strong>do not</strong> wrap the TopBar or Sidebar components.
```

Comment:
Should the Sidebar component have a doc page?

Reply:
Listed in issue #753